notebooks/5_add_columns.ipynb

```diff
@@ -0,0 +1,727 @@
+{
```

Comment:
<p>typo: "row labels as" should be "row labels and"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>